rhc-ose-ansible/inventory/ose-provision

```diff
@@ -55,7 +55,14 @@ public_dns_forwarder="8.8.8.8"
 
 rhc_ose_inv_dest=.
 #rhc_ose_ssh_user=root
-#rhc_ose_identity_provider=htpasswd
+
+### Set up Authentication for Open Shift
+# Example: htpasswd
+#rhc_ose_identity_provider=[{'name': 'htpasswd_auth', 'login': 'true', 'challenge': 'true', 'kind': 'HTPasswdPasswordIdentityProvider', 'filename': '/etc/origin/master/htpasswd'}]
+# Example: LDAP
+#rhc_ose_identity_provider=[{'name':'paas_ldap_provider','login':'true','challenge':'true','kind':'LDAPPasswordIdentityProvider','attributes':{'id':['dn'],'email':['mail'],'name':['displayName'],'preferredUsername':['uid']},'insecure':'true','bindDN':'uid=ldap,cn=users,cn=compat,dc=org,dc=example,dc=com','bindPassword':'password','url':'ldap://ldap.example.com:636/dc=org,dc=example,dc=com?uid?sub?(memberOf=cn=openshift,cn=groups,cn=accounts,dc=org,dc=example,dc=com)'}]
```

Comment:
minor nitpick - the example shows "insecure=true" and "ldap", but the port is 636 ... would be better if the example had port 389 since it's insecure.